pom.xml

```diff
@@ -20,7 +20,7 @@
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<java.version>1.8</java.version>
 		<spring-cloud.version>Edgware.SR2</spring-cloud.version>
-		<spring-cloud-function.version>1.0.0.M5</spring-cloud-function.version>
+		<spring-cloud-function.version>1.0.0.BUILD-SNAPSHOT</spring-cloud-function.version>
```

Comment:
Wouldn't depending on a snapshot version make the build non-reproducible?

Reply:
Yes, but that is a fact of life. We need the snapshot to pull in new features. I'm not going to change it.

Reply:
I gather the Java tooling will ensure that we don't ship a release dependent on a snapshot, so that's fine.